data_encryption/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright <2019> Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Encryption data",
+    "summary": "Store accounts and credentials encrypted by environment",
+    "version": "12.0.0.0.1",
```

Comment:
I'd use 1.0.0 + alpha dev status

Reply:
You are right, it could be nice to use alpha status for now. Will do